src/kolibri.yaml

```diff
@@ -0,0 +1,35 @@
+---
+description: >
+  The offline app for universal education.
+  Kolibri makes high quality education technology available in low-resource
+  communities such as rural schools, refugee camps, orphanages, non-formal
+  school systems, and prison systems.
+license: MIT
+name: Kolibri
+programmingLanguages:
+  - python
+  - javascript
+rating: 7
+repoUrl: "https://github.com/learningequality/kolibri"
+sdgs: [4]
+websiteUrl: "https://learningequality.org/kolibri/"
+
+contributionGuidelinesUrl: "https://kolibri-dev.readthedocs.io/en/develop/"
+logoUrl: "https://kolibri-dev.readthedocs.io/en/develop/_static/logo.png"
+
+ratingComment: Comprehensive documentation, high-looking project quality.
+otherLinks:
+  - name: Kolibri Demo
+    link: http://kolibridemo.learningequality.org/en/learn/#/topics
```

Comment:
We should document that `description` is optional.
It make sense that it is, though. Just never thought about it